### DISCUSSION TITLE
ONNX PyPi credentials

### DISCUSSION THREAD

**Discussion by kevinch-nv** [onnx/onnx#5180] [Q&A]
Hi all, 

I'm prepping for the v1.6.0 release, and according to the (somewhat outdated) doc (https://github.com/onnx/onnx/blob/master/docs/OnnxReleases.md) there is an ONNX pypi account. Anyone here know what the credentials are?

**Comment by prasanthpul** [comment]
they are not published broadly as we dont have a good mechanism for protecting. i had sent you mail with the info
